lib/modules/newCommentCount.js

```diff
@@ -40,6 +40,19 @@ module.options = {
 		value: true,
 		description: 'Show the Subscribe button?',
 	},
+	monitorPostsVisited: {
+		type: 'boolean',
+		value: true,
+		description: 'Monitor the number of comments on posts you have visited.',
+		advanced: true,
+	},
+	monitorPostsVisitedIncognito: {
+		dependsOn: 'monitorPostsVisited',
+		type: 'boolean',
+		value: false,
```

Comment:
I kinda prefer keeping this consistent across platforms so as to not catch multi-platformers by surprise, even with Chrome's disclaimer.